src/types/index.ts

```diff
@@ -150,14 +150,14 @@ export interface Subscription extends ProductCommon {
 
 export interface RequestPurchase {
   sku: Sku;
-  andDangerouslyFinishTransactionAutomaticallyIOS: boolean;
+  andDangerouslyFinishTransactionAutomaticallyIOS?: boolean;
```

Comment:
Looks like this is pretty urgent

Reply:
yeah, it must be throwing a lot of eslint errors on folks' projects since it has been introduced in an earlier version and these props were optional before